packages/babel-plugin-emotion/src/index.js

```diff
@@ -144,21 +144,40 @@ const getComponentId = (state, prefix: string = 'css') => {
 export function buildStyledCallExpression(identifier, tag, path, state, t) {
   const identifierName = getIdentifierName(path, t)
 
-  if (state.extractStatic && !path.node.quasi.expressions.length) {
+  if (state.extractStatic) {
     const { name, hash, src } = inline(
       path.node.quasi,
       identifierName,
       'styled' // we don't want these styles to be merged in css``
     )
 
     const cssText = `.${name}-${hash} { ${src} }`
-    const { staticCSSRules } = parseCSS(cssText, true, getFilename(path))
+    const { staticCSSRules, composesCount } = parseCSS(
+      cssText,
+      true,
+      getFilename(path)
+    )
+
+    // Help Needed:
+    // We should read the browser preferences for postcss
+    // and turn this on only if css vars are supported in their browser targets.
+    const objs = path.node.quasi.expressions.slice(0, composesCount)
+
+    if (objs.length) {
+      throw new Error('Cannot use composes in extractStatic mode.')
```

Comment:
@tkh44 Do you think we should just go to inline mode instead of throwing an error. We also need to check if there are any interpolations in other places before going to extract.